keyboards/cantor/keymaps/zeromist/keymap.c

```diff
@@ -0,0 +1,139 @@
+// Copyright 2022 Diego Palacios (@diepala)
+// SPDX-License-Identifier: GPL-2.0
+
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    MCOPY = SAFE_RANGE,
+    MPASTE,
+    MLCKSCN,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case MCOPY:
+        if (record->event.pressed) {
+            // Macro for copy
+            SEND_STRING(SS_LCTL("c"));
```

Comment:
you can use `C(KC_C)` in yur keymap, instead of this.

Reply:
So this one would look like this SEND_STRINGS(C(KC_C)) ?

Reply:
No.  I mean, you can put `C(KC_C)` directly into your keymap.  No need for a macro at all.